src/routes/blog/post/announcing-database-ai-suggestions/+page.markdoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Announcing Database AI suggestions: From table name to schema in one click"
+description: We’re introducing new AI features into Appwrite Databases that generate complete schemas and indexes from just a table name.
+date: 2025-10-01
+cover: /images/blog/announcing-database-ai-suggestions/cover.png
+timeToRead: 5
+author: darshan-pandya
+category: announcement
+featured: false
+---
+In many development workflows, setting up a database schema is one of the first and often one of the slowest steps. Starting with a blank schema, juggling naming conventions, remembering which fields need indexes, or ensuring every collection follows best practices can quickly become a chore.
+
+Whether spinning up a new project, designing an operational table for a feature, or standardizing schemas across multiple services, the setup should be fast, consistent, and reliable.
+
+That’s why we’re introducing **Database AI suggestions.** A simple but powerful way to generate sensible columns and indexes the moment you create a new table. By analyzing the table name, sibling tables, and any extra context you provide, Appwrite suggests a complete schema that you can tweak and apply in seconds. No more blank slates, no more forgotten indexes, and no more inconsistent naming.
+
+# Design schemas in seconds, not hours
+
+When creating a new table in the Appwrite Console, you can **choose to generate AI-powered suggestions** for columns and indexes tailored to your use case. For example, if you name your table `Orders`, Appwrite will suggest fields like `orderId`, `userId`, `totalAmount`, and `status`, complete with sensible types and recommended indexes.
+
+These suggestions are generated using a mix of:
+
+- The table’s **name** (e.g., “Orders”, “UserSessions”)
+- **Sibling tables and their respective columns** in your database to align naming and structure
+- Any **additional context** you provide in your prompt
+
+You’re always in control. Suggestions don’t apply automatically. You can review them, adjust inline, or skip them entirely. Once ready, applying the schema is just one click, including an optional CTA to add suggested indexes for even better performance.
+
+Alongside Database AI Suggestions, we have also added an **AI-powered index suggestions UI.** A new desktop modal and mobile side-sheet offer guided flows with toggles and an easy “apply” step for indexes.
```

Comment:
not sure if modal and side-sheet are something to mention, pretty standard across console.